dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -27,7 +27,7 @@ export interface IAppUpgradeProps {
     values?: string,
     schema?: JSONSchema4,
   ) => Promise<boolean>;
-  fetchChartVersions: (namespace: string, id: string) => Promise<IChartVersion[]>;
+  fetchChartVersions: (namespace: string, id: string) => Promise<any>;
```

Comment:
Interested to test why this is needed too (though I trust that it is).

Reply:
well, actually is not any, but it could be undefined